austation/code/game/objects/structures/party_balloons.dm

```diff
@@ -0,0 +1,132 @@
+/////////////////////  CONSUMABLE BALLOON  /////////////////////
+
+
+/obj/item/balloon
+	name = "deflated balloon"
+	desc = "A small, deflated balloon. You can use it in your hand to inflate it."
+	icon = 'austation/icons/obj/structures/balloons.dmi'
+	icon_state = "red"
+	w_class = WEIGHT_CLASS_TINY
+	var/coloring = "red"
+	var/inflate_time = 20
+	var/static/list/colorslist = list(
+		"red",
+		"blue",
+		"green",
+		"yellow",
+		"purple",
+		"pink"
+	)
+
+/obj/item/balloon/Initialize()
+	. = ..()
+	coloring = pick(colorslist)
+	icon_state = coloring
```

Comment:
I'm not sure what the point of the colouring variable is. It just seems like an un-needed intermediary.
icon_state can just be set directly
```suggestion
	icon_state = "red"
	w_class = WEIGHT_CLASS_TINY
	var/inflate_time = 20
	var/static/list/colorslist = list(
		"red",
		"blue",
		"green",
		"yellow",
		"purple",
		"pink"
	)

/obj/item/balloon/Initialize()
	. = ..()
	icon_state = pick(colorslist)
```

Reply:
the reason for the coloring variable is so we can pass the same colour onto the structure when it is spawned.